apps-rendering/src/components/Callout/styles.ts

```diff
@@ -0,0 +1,218 @@
+import type { SerializedStyles } from '@emotion/react';
+import { css } from '@emotion/react';
+import { text } from '@guardian/common-rendering/src/editorialPalette';
+import type { ArticleFormat } from '@guardian/libs';
+import {
+	body,
+	brandAlt,
+	error,
+	headline,
+	neutral,
+	remSpace,
+	success,
+	until,
+} from '@guardian/source-foundations';
+import { darkModeCss } from 'styles';
+
+// Highlight Component Styles
+export const highlightStyles = css`
+	color: ${neutral[7]};
+	display: flex;
+	align-items: center;
+	padding: 0 ${remSpace[1]};
+	font-family: GuardianTextSans;
+	background: ${brandAlt[400]};
+
+	${darkModeCss`
+		background: ${brandAlt[200]};
+	`}
+`;
+
+// Callout block styles
+export const calloutContainerStyles = (
+	format: ArticleFormat,
+): SerializedStyles => css`
+	padding-bottom: ${remSpace[12]};
+	background: ${neutral[97]};
+	color: ${neutral[7]};
+	a {
+		color: ${text.calloutPrimary(format)};
+	}
+
+	${darkModeCss`
+		background: ${neutral[20]};
+		color: ${neutral[86]};
+		a {
+			color: ${neutral[86]};
+	`}
+`;
+export const calloutDetailsStyles = css`
+	padding-bottom: ${remSpace[6]};
+
+	/* IE does not support summary HTML elements, so we need to hide children ourself */
+	:not([open]) > *:not(summary) {
+		display: none;
+	}
+`;
+
+export const calloutSummaryStyles = css`
+	/* Remove default styles from summery tag */
+	::-webkit-details-marker {
+		display: none;
+	}
+	outline: none;
+
+	/* We don't want the summary to open when we click anything but the button, so we pointer-event: none the summary */
+	pointer-events: none;
+
+	/*  why hide visibility?
+        We want to prevent the user for tabbing to the summery HTML element
```

Comment:
Is all this accessibility stuff still needed since you did your work on the expanding wrapper etc? Would be useful to know for my PR too.

Reply:
Yeah good spot - I've removed it